### PR TITLE
Pinning now works as expected

### PR DESCRIPTION
**Pull request by annando**
Other should be able seeing your pinned posts and there are icons for all themes.

### CODE REVIEW

File: src/Module/Profile.php [hunk 1]
```diff
@@ -350,7 +350,7 @@ public static function content(array $parameters = [], $update = 0)
 		$items = DBA::toArray($items_stmt);
 
 		if ($pager->getStart() == 0 && !empty($a->profile['profile_uid'])) {
-			$pinned_items = Item::selectPinned($a->profile['profile_uid'], ['uri']);
+			$pinned_items = Item::selectPinned($a->profile['profile_uid'], ['uri', 'pinned'], ['true' . $sql_extra]);
```
Comment (MrPetovan) [inline src/Module/Profile.php]: Is the `true` necessary?

Comment (annando) [reply src/Module/Profile.php]: Yeah. The variable `$sql_extra` does contain stuff like ` AND NOT private`. So we have to work around that `AND`.

**Comment by annando** [comment]
Hopefully this had been my last change to the profile.php.

**Comment by AlfredSK** [comment]
Hmm. Still not working here. Or I didn't understand how it is supposed to work.
I pinned a post. But when I look at my profile page (example.com/profile/$user) there is no pinned post at the top of the page.

Maybe related? https://github.com/friendica/friendica/issues/7676#issuecomment-552008088

**Comment by annando** [comment]
Yeah, well, With PR https://github.com/friendica/friendica/pull/7826 it should work :-)

**Pull request merged**